index.Rmd

```diff
@@ -6,13 +6,13 @@ output:
     keep_md: true
 ---
 
-# Introduction
```

Comment:
I'm curious why your introduction and data and methods show up in scrollable boxes. I think it's a little hard to read and you would benefit from making them just static text.